compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -513,7 +513,17 @@ bool ConvertNCHWToNHWCPass::run(loco::Graph *g)
     {
       // List of supported Ops
       case luci::CircleOpcode::CIRCLEINPUT:
+        if (!_preserve_input && !has_data_format(node))
```

Comment:
Q) whay also check `has_data_format` here? and if the input node doesn't have data format why set to `NCHW`?

Reply:
ConverNCHWToNHWC works as below.

1. Set data format of target nodes to NCHW (**this should be done once**)
2. Convert the target nodes from NCHW  to NHWC.

The above steps are repeated until no change is detected. To do step 1 once, I check the existence of data_format.

Reply:
Thanks! I've forgot -_-;